src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1000,6 +1007,24 @@ class DocBaseViewer extends BaseViewer {
             // Add page IDs to each page after page structure is available
             this.setupPageIds();
         }
+
+        this.initThumbnails();
+    }
+
+    initThumbnails() {
+        this.thumbnailsSidebar.init({
+            totalItems: this.pdfViewer.pagesCount,
+            itemHeight: 115,
+            containerHeight: this.docEl.clientHeight,
+            marginTop: 7.5,
```

Comment:
Is this the pixel value in the design? Can you pass in a class and allow these to live in CSS, instead?